8.cytopick_analysis/scripts/feature_median_image_find.py

```diff
@@ -0,0 +1,363 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import cv2
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+# import pillow and open cv
```

Comment:
Recommend including open cv import under this code comment instead of it being above. Or remove from code comment.

```suggestion
# import pillow and open cv
import cv2
```